### PR TITLE
fix: Added a fixed uksouth location for `avm.res.dev-ops-infrastructure.pool`

### PR DESCRIPTION
**Pull request by johnlokerse**
## Description

This pull request introduces a fixed location (uksouth) used to run the validation deployments.

## Pipeline Reference

| Pipeline |
| -------- |
|     [avm.res.dev-ops-infrastructure.pool](https://github.com/johnlokerse/bicep-registry-modules/actions/workflows/avm.res.dev-ops-infrastructure.pool.yml)     |

## Type of Change

- [x] Update to CI Environment or utilities (Non-module affecting changes)
- [ ] Azure Verified Module updates:
  - [ ] Bugfix containing backwards-compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] I have run `Set-AVMModule` locally to generate the supporting module files.
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings

### CODE REVIEW

File: avm/res/dev-ops-infrastructure/pool/tests/e2e/defaults/main.test.bicep
[the remark this answers is not in the document]
Comment (johnlokerse) [reply avm/res/dev-ops-infrastructure/pool/tests/e2e/defaults/main.test.bicep:1]: Done @eriqua 😃

File: avm/res/dev-ops-infrastructure/pool/README.md [hunk 1]
```diff
@@ -559,7 +559,7 @@ module pool 'br/public:avm/res/dev-ops-infrastructure/pool:<version>' = {
 | Parameter | Type | Description |
 | :-- | :-- | :-- |
 | [`diagnosticSettings`](#parameter-diagnosticsettings) | array | The diagnostic settings of the service. |
-| [`enableTelemetry`](#parameter-enabletelemetry) | bool | Enable/Disable usage telemetry for module. |
+| [`enableTelemetry`](#parameter-enabletelemetry) | bool | Enable or disable telemetry for the module. |
```
Comment (eriqua) [inline avm/res/dev-ops-infrastructure/pool/README.md:562]: Sorry for the back and forth. This update makes sense, but I'd avoid it since it's a shared parameter, used with same naming and description by all modules.

I'd suggest to revert this change and actually no other change is needed. Since bicep version got updated, we have a change already in main.json and that alone qualifies for publishing

Comment (johnlokerse) [reply avm/res/dev-ops-infrastructure/pool/README.md]: Reverted and ran Set-AVMModule. Good to know for next time :)

**Review by eriqua — APPROVED** [review]
🆗

**Comment by johnlokerse** [comment]
> 🆗

@eriqua Do you know when this PR gets merged? 😃
